app/helpers/vacancies_helper.rb

```diff
@@ -66,28 +66,42 @@ def vacancy_about_school_value(vacancy)
     vacancy_or_organisation_description(vacancy)
   end
 
+  def vacancy_job_location_summary(vacancy)
```

Comment:
I think it could be a good idea to create a `JobLocationHelper` class at this point. This helper is 170 lines long and we've got a few helpers exclusively related to job location.

What do you think?